cmake/tpl/SundialsPETSC.cmake

```diff
@@ -49,13 +49,14 @@ endif()
 # Section 3: Find the TPL
 # -----------------------------------------------------------------------------
 
-find_package(PETSC REQUIRED)
+find_package(PETSC 3.5.0 REQUIRED)
 
-message(STATUS "PETSC_DIR:         ${PETSC_DIR}")
-message(STATUS "PETSC_LIBRARIES:   ${PETSC_LIBRARIES_}")
-message(STATUS "PETSC_INCLUDES:    ${PETSC_INCLUDES_}")
-message(STATUS "PETSC_INDEX_SIZE:  ${PETSC_INDEX_SIZE}")
-message(STATUS "PETSC_PRECISION:   ${PETSC_PRECISION}\n")
+message(STATUS "PETSC_DIR:          ${PETSC_DIR}")
+message(STATUS "PETSC_ARCH:         ${PETSC_ARCH}")
+message(STATUS "PETSC_LIBRARIES:    ${PETSC_LIBRARIES}")
+message(STATUS "PETSC_INCLUDE_DIRS: ${PETSC_INCLUDE_DIRS}")
+message(STATUS "PETSC_INDEX_SIZE:   ${PETSC_INDEX_SIZE}")
+message(STATUS "PETSC_PRECISION:    ${PETSC_PRECISION}")
```

Comment:
Are `PETSC_INDEX_SIZE` and `PETSC_PRECISION` still defined? I think these got removed in the switch to pkgconfig. Can we still get these to check compatibility?

Reply:
They are not defined, and actually havent been getting populated in awhile from what I can tell. PETSc doesnt put the information in the files we currently/used to look in. We will have to do something else instead. I left them here for now and they will just continue to be unpopulated.